siliconcompiler/schema/schema_cfg.py

```diff
@@ -259,6 +247,18 @@ def schema_fpga(cfg):
             tool. The part name must be an exact string match to the partname
             hard coded within the FPGA EDA tool.""")
 
+    scparam(cfg, ['fpga', family, 'lutsize'],
+            sctype='str',
+            defvalue='4',
+            require='fpga',
+            shorthelp="FPGA: lutsize",
+            switch="-fpga_lutsize <str>",
```

Comment:
cli switch signature should be "-fpga_lutsize \<family\> \<int\>"

Reply:
Done